packages/vue/tsconfig.json

```diff
@@ -1,12 +1,20 @@
 {
   "extends": "@aws-amplify/typescript-config/tsconfig.vue.json",
   "include": [
-    "__mocks__",
-    "__tests__",
+    "src/**/*",
+    "types/**/*",
+    "*.d.ts",
     "*.ts",
-    "src/**/*.ts",
-    "src/**/*.tsx",
-    "src/**/*.vue"
+    "__tests__/**/*",
```

Comment:
Other package tsconfigs don't require this line for type checking the entry point _\_\_tests\_\__ dir, why is it necessary here?